content/faq/content.md

```diff
@@ -0,0 +1,34 @@
+---
+title: What kind of content can I upload, and what about illegal or infringing content?
+category: publisher
+---
+
+This guide provides answers about questions regarding what you may upload to the LBRY app and network, as well as how to report illegal or infringing content.
+
+###What content can I legally upload to LBRY?
+
+You may upload content you created or own that does not infringe on the rights or safety of others, or content you have permission to upload. This includes Public Domain content, Creative Commons or Open Source content shared according to the license requirements, and Copyrighted content where you have permission from the owner.
+
+###Isn’t LBRY decentralized and the content can’t be removed?
+
+It is important to make a distinction between the LBRY blockchain/protocol and any applications running on top of them when referring to censorship. The LBRY protocol is fully decentralized and censorship resistant, with the protocol storing metadata and naming on the blockchain, and facilitating data transfers over a peer to peer (P2P) network. However, LBRY also makes an app to demonstrate our protocol. With our app we will engage in non-arbitrary censorship, meaning only horrific or infringing content will be removed. As a U.S. company LBRY Inc. and our app will follow all U.S. laws including DMCA. If someone made an app or website using the LBRY protocol in some other country, it would have to follow that country's laws, which aren’t necessarily the same as ours. Either app would read the same blockchain, though. 
```

Comment:
- The LBRY protocol encompasses both the blockchain and data network. I don't think the slash is necessary.
- I'd paragraph break before the however. But I also have a preference for short paragraphs. 
- Mention CDA along with DMCA
- Definitely mention that LBRY makes no strong guarantees that data will be hosted. If no one is willing to host data, it won't continue to be available on the network.